__tests__/app.test.js

```diff
@@ -120,3 +120,50 @@ describe("5.GET /api/articles/:article_id", () => {
       });
   });
 });
+describe("6.GET /api/articles/:article_id/comments", () => {
+  test("responds with statusCode 200 and the response contains an array of comments", () => {
+    return request(app)
+      .get("/api/articles/3/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toBeInstanceOf(Array);
+        expect(comment).toHaveLength(2);
+        comment.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            })
+          );
+        });
+      });
+  });
```

Comment:
Please include another test where the id exists, but there are no comments for that article with that id. This should expect a status 200 and an empty array.